src/test/groovy/org/prebid/server/functional/tests/ImpRequestSpec.groovy

```diff
@@ -156,6 +157,34 @@ class ImpRequestSpec extends BaseSpec {
         bidderName << [WILDCARD, UNKNOWN]
     }
 
+    def "PBS shouldn't update imp fields without warning when imp.ext.prebid.imp contain not applicable bidder"() {
```

Comment:
PBS shouldn't update imp fields and without warning when imp.ext.prebid.imp contain not applicable bidder